tools/hidl/hidlcomp.cpp

```diff
@@ -399,11 +399,7 @@ void validateProfileExecutionOptions(std::string &options)
     // Expected format:
     //   options     ::= ("s" | "ms" | "us" | "ns") "," <bucketLimit> ["," <bucketLimit>]*
     //   bucketLimit ::= digit+
-    //
-    // No blanks allowed
-
-    if (options.find_first_of(' ') != std::string::npos)
-        throw "Execution profiling option may not include blanks";
+    // Note, caller must remove any spaces
```

Comment:
This is correct, but this wasn't exactly the intention of my comment.  Caller must remove spaces in order for there to be consistency, not just to pass this test.

Your syntax accepts spaces, it's an internal detail that they are stripped out before use.  If another place calls this validate and it doesn't work because spaces aren't removed then it will be confusing for users.